sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md

```diff
@@ -47,6 +47,49 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
 CommunicationServiceResource communicationService = communicationServiceLro.Value;
 ```
 
+***Create a Communication Service with User Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with User Assigned Managed Identity
+CommunicationServiceResourceCollection collection = resourceGroup.GetCommunicationServiceResources();
+string communicationServiceName = "myCommunicationService";
+string userAssignedIdentityResourceId = "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{userAssignedIdentityName}";
+
+var identity = new CommunicationServiceIdentity(
+    type: IdentityType.UserAssigned,
+    userAssignedIdentities: new Dictionary<string, CommunicationServiceUserAssignedIdentities>()
+    {
+        { userAssignedIdentityResourceId, new CommunicationServiceUserAssignedIdentities() }
+    });
+
+CommunicationServiceResourceData data = new CommunicationServiceResourceData("global")
+{
+    DataLocation = "UnitedStates",
+    Identity = identity
+};
+
+ArmOperation<CommunicationServiceResource> communicationServiceLro = await collection.CreateOrUpdateAsync(WaitUntil.Completed, communicationServiceName, data);
+CommunicationServiceResource communicationService = communicationServiceLro.Value;
+
+
+```
+
+***Create a Communication Service with System Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with System Assigned Managed Identityy
+string communicationServiceName = "myCommunicationService";
+CommunicationServiceResourceCollection collection = resourceGroup.GetCommunicationServiceResources();
+
+CommunicationServiceResourceData data = new CommunicationServiceResourceData("global")
+{
+    DataLocation = "UnitedStates",
+    Identity = new CommunicationServiceIdentity(type: IdentityType.SystemAssigned, userAssignedIdentities: null)
```

Comment:
I don't think you need to send in the userAssignedIdentities if you are assigned a system identity